recipes/zen-engine/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "zen-engine" %}
+{% set version = "0.8.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/zen_engine-{{ version }}.tar.gz
+  sha256: f43c08bdf67e088956fdd46fe07d7a8a65939c42fc971deb421345eb81f69917
+
+build:
+  entry_points:
+    - zenengine = reader.__main__:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - maturin >=0.14,<0.15
+    - pip
+    - rust
+  run:
+    - python >=3.7
+
+test:
+  imports:
+    - zen_engine
+  commands:
+    - pip check
+    - zenengine --help
+  requires:
+    - pip
+
+about:
+  home: https://gorules.io/
+  summary: Open-Source Business Rules Engine
+  license: MIT
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - CarinaPengQC
```

happy to be a co-maintainer. :)

```suggestion
    - CarinaPengQC
    - janjagusch
```